### PR TITLE
UIOR-1222: Update error message for opening PO with location-restricted funds

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[https://folio-org.atlassian.net/browse/UIOR-1222](https://folio-org.atlassian.net/browse/UIOR-1222) - Update error message for opening PO with location-restricted funds
## Approach

https://github.com/folio-org/ui-orders/assets/116072773/2ace0da2-9056-40d7-a552-8e71b660ea6c

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-orders/en.json [hunk 1]
```diff
@@ -164,7 +164,7 @@
   "errors.forbiddenDeleteSystemValues": "Can not delete system values",
   "errors.forbiddenDeleteUsedValue": "This value can not be deleted",
   "errors.fundCannotBePaid": "One or more fund distributions on this order can not be encumbered, because there is not enough money in <b>{fundCodes}</b>.",
-  "errors.fundLocationRestrictionViolation": "Unable to open PO. Fund ({fundCode}) is not valid with location ({locationCode})",
+  "errors.fundLocationRestrictionViolation": "Unable to open PO. One or more funds for POL number <b>{polNumber}</b> are not valid with the selected location(s)",
```
Comment (NikitaSedyx) [inline translations/ui-orders/en.json:167]: how is it connected to opening order?

Comment (alisher-epam) [reply translations/ui-orders/en.json:167]: The same error message. https://github.com/folio-org/ui-orders/pull/1555/files#diff-78be3eac36203c30b2072b3fe2be4c399302149888b622f22189ac28ebd67190R251

Comment (NikitaSedyx) [reply translations/ui-orders/en.json:167]: why do you have the same messages? `errors.openOrder.fundLocationRestrictionViolation` is for order open, how `errors.fundLocationRestrictionViolation` is connected to order open process?

Comment (alisher-epam) [reply translations/ui-orders/en.json]: Indeed, the error message has been used. I have removed it.

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**